go-console/.gitignore

```diff
@@ -0,0 +1,2 @@
+/target
```

Comment:
Using the `.gitignore` from this collection of templates on Github would be nice: https://github.com/github/gitignore/blob/main/Go.gitignore